src/clarity-angular/popover/dropdown/dropdown-toggle.ts

```diff
@@ -30,7 +30,7 @@ export class DropdownToggle {
     }
 
     get active(): boolean {
-        return this._dropdown.open;
+        return this._dropdown.ifOpenService.open;
     }
 
     @HostListener("click")
```

Comment:
I was wrong; it's using the getter so I've just edited it to use the injected service.